panda_gym/pybullet.py

```diff
@@ -19,24 +18,17 @@ class PyBullet:
         n_substeps (int, optional): Number of sim substep when step() is called. Defaults to 20.
         background_color (np.ndarray, optional): The background color as (red, green, blue).
             Defaults to np.array([223, 54, 45]).
-        render (bool, optional): Deprecated: This argument is deprecated and will be removed in a future
-            version. Use the render_mode argument instead.
+        renderer (str, optional): Renderer, either "Tiny" or OpenGL". Default to "Tiny".
```

Comment:
Maybe mention relation to `render_mode` here as well. For 'human', render_mode is ignored/defaults to OpenGL.